pkg/agent/openflow/pipeline.go

```diff
@@ -1096,7 +1094,6 @@ func (c *client) allowRulesMetricFlows(conjunctionID uint32, ingress bool) []bin
 	metricFlow := func(isCTNew bool, protocol binding.Protocol) binding.Flow {
 		return c.pipeline[metricTableID].BuildFlow(priorityNormal).
 			MatchProtocol(protocol).
-			MatchPriority(priorityNormal).
```

Comment:
@wenyingd : I assume MatchPriority() is not needed?

Reply:
I think we could remove the call for `MatchPriority()` in the functions in this file (pipeline.go), but please note that we should keep it in file (network_policy.go), for CNP might call this API to reset flow priority. An originally discussion is https://github.com/vmware-tanzu/antrea/pull/803#discussion_r441752091

Reply:
Got it.